core/internal/cluster/kafka_cluster.go

```diff
@@ -98,6 +103,19 @@ func (module *KafkaCluster) Start() error {
 	// Start main loop that has a timer for offset and topic fetches
 	module.offsetTicker = time.NewTicker(time.Duration(module.offsetRefresh) * time.Second)
 	module.metadataTicker = time.NewTicker(time.Duration(module.topicRefresh) * time.Second)
+
+	if module.groupsReaperRefresh != 0 {
+		module.groupsReaperTicker = time.NewTicker(time.Duration(module.groupsReaperRefresh) * time.Second)
```

Comment:
`groupsReaperRefreshInSeconds `

Reply:
I'm following the current code conventions, `offsetRefresh` and `topicRefresh` don't have seconds as part of the naming